camera_matrix:
    # ordered as: Fx, Fy, Cx, Cy
    cam_mat: [4.558675e+02, 4.569925e+02, 3.633525e+02, 1.724759e+02]
    # ordered as: K1, K2, K3, K4, K5, K6 (assumed as 0, not in code)
    dist_coeffs: [-2.994090e-01, 9.807324e-02, -2.577654e-03, -9.520364e-04, 0.0, 0.0]
    # assuming no tangential distortion (p1, p2 = 0)
